src/float/printer/ieee.cr

```diff
@@ -109,8 +109,6 @@ module Float::Printer::IEEE
     physical_significand_is_zero = (d64 & SIGNIFICAND_MASK_64) == 0
 
     lower_bound_closer = physical_significand_is_zero && (exponent(d64) != DENORMAL_EXPONENT_64)
-    calcualted_exp = exponent(d64)
```

Comment:
This needs some review to check if there isn't somethings missing

Reply:
No offense intended, but I feel like this kind of comment is likely to stall a lot of useful PRs because it suggests that there are problems but doesn't move us closer to resolving such problems.
If you suspect there really is a problem but are unsure, will you please tag someone who has experience in that part of the codebase, so we can progress the PR? Otherwise it will just go stale and we'll end up losing useful contributions (and motivation).